### DISCUSSION TITLE
upgrading bom 0.21.0 → 0.22.1 missing ai.djl.pytorch:pytorch-native-cpu:jar:osx-x86_64

### DISCUSSION THREAD

**Discussion by nezda** [deepjavalibrary/djl#2597] [Q&A]
Hello, all -

My code included the bom for `0.21.0`

```xml
    <dependencyManagement>
      <dependencies>
        <dependency>
          <groupId>ai.djl</groupId>
          <artifactId>bom</artifactId>
          <version>0.21.0</version>
          <type>pom</type>
          <scope>import</scope>
        </dependency>
      </dependencies>
    </dependencyManagement>
```

and dependency

```xml
        <dependency>
          <groupId>ai.djl.pytorch</groupId>
          <artifactId>pytorch-native-cpu</artifactId>
          <classifier>osx-x86_64</classifier>
          <scope>runtime</scope>
        </dependency>
```

When I upgraded bom from 0.21.0 → 0.22.1 I got mvn error `dependencies.dependency.version' for ai.djl.pytorch:pytorch-native-cpu:jar:osx-x86_64 is missing`. When I pinned the version of that dependency to `<version>1.13.1</version>` my code works. Is this a problem with the bom? Is there no 2.0 version compiled for `osx-x86_64` for `pytorch-native-cpu`?

Please advise,
\- Luke

**Comment by frankfliu** [comment]
Thanks for reporting this bug. Will look into and fix the bug.
